CMakeLists.txt

```diff
@@ -29,8 +29,8 @@ list(APPEND CMAKE_MODULE_PATH ${BCOS_FRAMEWORK_CMAKE_DIR})
 # init hunter
 include(HunterGate)
 HunterGate(
-    URL "https://github.com/FISCO-BCOS/hunter/archive/1984be9381f472aec11307f009344ef57b3cb4db.tar.gz"
-    SHA1 "3442c3ba973f505daec46ae7a42f9a2429e433f9"
+    URL "https://github.com/cyjseagull/hunter/archive/5314cbfa19b5527c75c5ab0c1f57c5138097aa9a.tar.gz"
```

Comment:
FISCO-BCOS maybe better

Reply:
Will switch to FISCO BCOS  when this PR merged: https://github.com/FISCO-BCOS/hunter/pull/9